Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -217,9 +217,17 @@ open class XCTWaiter {
         self.manager = manager
 
         // Begin the core wait loop.
+#if os(Windows)
+        let timeoutTimestamp = NSDate._currentAbsoluteTime() + timeout
+#else
         let timeoutTimestamp = CFAbsoluteTimeGetCurrent() + timeout
+#endif
         while !isFinished {
+#if os(Windows)
+            let remaining = timeoutTimestamp - NSDate._currentAbsoluteTime()
+#else
             let remaining = timeoutTimestamp - CFAbsoluteTimeGetCurrent()
```

Comment:
`let remaining = timeoutTimestamp - Date.timeIntervalSinceReferenceDate`

Reply:
lol, that is way better, and allows us to kill a SPI, I like it!